swagger/v1/timetable_entry.yaml

```diff
@@ -0,0 +1,42 @@
+TimetableEntry:
+  type: object
+  required:
+    - id
+    - type
+    - attributes
+  properties:
+    id:
+      type: string
+      format: uuid
+      example: f0a91e16-1b0e-4e1f-93fe-319dda9933e6
+      description: The unique identifier (UUID) of this object
+    type:
+      type: string
+      example: timetable
+      enum:
+        - timetable
+      description: The type of this object - always `timetable`
+    attributes:
+      type: object
+      required:
+        - start_time
+        - nomis_type
+        - reason
+      properties:
+        start_time:
+          type: string
+          example: "2020-04-22T08:30:00"
```

Comment:
might be sensible to force a timezone on time parameters ? (though I know there are timezone problems further downstream in Nomis)

Reply:
Oh good spot. So this is actually just an iso8601 time stamp so I'll update this. We infer it as London so will make it match that